resources/public/api/conf/1.0/employment_expenses_create_and_amend.yaml

```diff
@@ -22,7 +22,11 @@ description: |
           <tr>
               <td><p>NOT_FOUND</p></td>
               <td><p>Scenario to simulate not found error.</p></td>
-           </tr>
+          </tr>
+          <tr>
+              <td><p>STATEFUL</p></td>
+              <td><p>Performs a stateful create or update.</p></td>
```

Comment:
This one needs double checking with content writer. As its not specified in the doc

Reply:
Ben has confirmed this description is fine

Reply:
Nice Thanks!